src/OpenRoad.cc

```diff
@@ -396,6 +398,27 @@ OpenRoad::linkDesign(const char *design_name)
   }
 }
 
+void 
+OpenRoad::partitionDesign(unsigned int max_num_macro, unsigned int min_num_macro,
+                          unsigned int max_num_inst,  unsigned int min_num_inst,
```

Comment:
really bad practice to use unsigned. the first time you subtract them and end up with a negative values you will see  why